FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/SendPasswordProtectedMessageFlowTest.kt

```diff
@@ -0,0 +1,182 @@
+    onView(withId(R.id.floatActionButtonCompose))
+      .check(matches(isDisplayed()))
+      .perform(click())
+
+    onView(withId(R.id.chipLayoutTo))
+      .perform(scrollTo())
+    onView(withId(R.id.editTextEmailAddress))
+      .perform(
+        typeText(RECIPIENT_WITHOUT_PUBLIC_KEY),
+        pressImeActionButton(),
+        closeSoftKeyboard()
+      )
+    onView(withId(R.id.editTextEmailSubject))
+      .perform(
+        scrollTo(),
+        click(),
+        typeText(MESSAGE_SUBJECT),
+        closeSoftKeyboard()
+      )
+    onView(withId(R.id.editTextEmailMessage))
+      .perform(
+        scrollTo(),
+        typeText(MESSAGE_TEXT),
+        closeSoftKeyboard()
+      )
+
+    for (att in atts) {
+      addAtt(att)
+    }
+
+    onView(withId(R.id.btnSetWebPortalPassword))
+      .perform(
+        scrollTo(),
+        click()
+      )
+
+    onView(withId(R.id.eTPassphrase))
+      .perform(
+        typeText(WEB_PORTAL_PASSWORD),
+        pressImeActionButton()
+      )
+
+    onView(withId(R.id.menuActionSend))
+      .check(matches(isDisplayed()))
+      .perform(click())
+
+    Thread.sleep(10000)
+  }
+
+  companion object {
+    private const val RECIPIENT_WITHOUT_PUBLIC_KEY = "no_key@flowcrypt.test"
+    private const val WEB_PORTAL_PASSWORD = "Qwerty1234@"
+    private const val MESSAGE_SUBJECT = "Subject"
+    private const val MESSAGE_TEXT = "Some text"
+    private const val ATTACHMENT_NAME_1 = "text.txt"
+    private const val ATTACHMENT_NAME_2 = "text1.txt"
+    private const val ATTACHMENT_NAME_3 = "binary_key.key"
+    private const val REPLY_TOKEN = "some_reply_token"
+    private var atts: MutableList<File> = mutableListOf()
+    private val pgpSecretKeyRing = PGPainless.generateKeyRing().simpleEcKeyRing(
+      UserId.nameAndEmail(RECIPIENT_WITHOUT_PUBLIC_KEY, RECIPIENT_WITHOUT_PUBLIC_KEY),
+      TestConstants.DEFAULT_PASSWORD
+    )
+
+    @BeforeClass
+    @JvmStatic
+    fun setUp() {
+      val directory = InstrumentationRegistry.getInstrumentation().targetContext
+        .getExternalFilesDir(Environment.DIRECTORY_DOCUMENTS)
+        ?: File(Environment.DIRECTORY_DOCUMENTS)
+      atts.add(
+        TestGeneralUtil.createFileWithContent(
+          directory = directory,
+          fileName = ATTACHMENT_NAME_1,
+          inputStream = "Text for filling the attached file".toInputStream()
+        )
+      )
+
+      atts.add(
+        TestGeneralUtil.createFileWithContent(
+          directory = directory,
+          fileName = ATTACHMENT_NAME_2,
+          inputStream = "Text for filling the attached file".toInputStream()
+        )
+      )
+
+      val buffer = ByteArrayOutputStream()
+      pgpSecretKeyRing.encode(buffer)
+
+      atts.add(
+        TestGeneralUtil.createFileWithContent(
+          directory = directory,
+          fileName = ATTACHMENT_NAME_3,
+          inputStream = ByteArrayInputStream(buffer.toByteArray())
+        )
+      )
+    }
```

Comment:
How about do like this?
```kt
val buffer = ByteArrayOutputStream()
pgpSecretKeyRing.encode(buffer)
atts = listOf(
  TestGeneralUtil.createFileWithContent(directory, ATTACHMENT_NAME_1, "Text for filling the attached file".toInputStream()),
  TestGeneralUtil.createFileWithContent(directory, ATTACHMENT_NAME_2, "Text for filling the attached file".toInputStream()),
  TestGeneralUtil.createFileWithContent(directory, ATTACHMENT_NAME_3, ByteArrayInputStream(buffer.toByteArray()))
) as MutableList<File>
```

Reply:
unfortunately, these lines are too long and will be formatted to 

```
val buffer = ByteArrayOutputStream()
      pgpSecretKeyRing.encode(buffer)
      atts = listOf(
        TestGeneralUtil.createFileWithContent(
          directory,
          ATTACHMENT_NAME_1,
          "Text for filling the attached file".toInputStream()
        ),
        TestGeneralUtil.createFileWithContent(
          directory,
          ATTACHMENT_NAME_2,
          "Text for filling the attached file".toInputStream()
        ),
        TestGeneralUtil.createFileWithContent(
          directory,
          ATTACHMENT_NAME_3,
          ByteArrayInputStream(buffer.toByteArray())
        )
      ) as MutableList<File>
```

Reply:
But anyway it looks better